utils/tables/ui_utils.R

```diff
@@ -210,7 +210,7 @@ tables_controlbar <- function(page, league_config, tables_rv) {
                 column(12,
                        h4("Player Settings"),
                        tables_cb_numeric(page, league_config, tables_rv,
-                                         "Minimum Minutes Played", "minimum_minutes", 25),
+                                         "Minimum Minutes Played", "minimum_minutes", 250),
```

Comment:
Can we revert all these back to 25? 250 minutes is too big a jump when using the keyboard keys to slide these values up and down.